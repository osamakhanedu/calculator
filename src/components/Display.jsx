import React from 'react';
import PropTypes from 'prop-types';

const Display = ({ displayValue }) => <div className="display-container">
<p className="display-value">
  {displayValue}
</p>
</div>;

Display.propTypes = { displayValue: PropTypes.string.isRequired };

export default Display;