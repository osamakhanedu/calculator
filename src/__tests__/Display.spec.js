import React from 'react';
import { shallow } from 'enzyme';
import Display from '../components/Display';


describe('Display Component', () => {

    let wrapper;
  
    beforeEach(() => wrapper = shallow(<Display displayValue={''} />));
  
    it('should render div',()=>{   
      expect(wrapper.find('div').length).toEqual(1);
    })
    
   
    
    
  });
  