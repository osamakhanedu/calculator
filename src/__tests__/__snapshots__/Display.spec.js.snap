// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Component should render correctly 1`] = `ShallowWrapper {}`;
